core/src/main/java/org/mskcc/cbio/portal/dao/DaoCancerStudy.java

```diff
@@ -34,6 +34,7 @@
 
 import java.sql.*;
 import java.text.*;
+import java.time.LocalDate;
```

Comment:
LocalDate is a very new class in the java SDK .. it requires a java 1.8 compiler. Do we have other code in the codebase which requires java 1.8? I think we do, so ok -- but if there is an easy 1.7 compatible class to accomplish the same thing, that might be wiser (in case other centers are using 1.7)